packages/orbit-components/src/ErrorForms.stories.tsx

```diff
@@ -27,12 +28,32 @@ const objectOptions = [
   { value: 3, label: "Third item" },
 ];
 
-export default {
+type ErrorFormsPropsAndCustomArgs = React.ComponentProps<typeof InputField> &
+  React.ComponentProps<typeof Select> & { showMore?: boolean; value: string | undefined };
+
+const meta: Meta<ErrorFormsPropsAndCustomArgs> = {
   title: "ErrorForms Kitchensink",
+
+  args: {
+    label: "Label",
+    help: "Is the spelling correct?",
+    error: "Something went wrong.",
+    value: "",
+    placeholder: "Placeholder",
+    prefix: "$",
+    required: true,
+    helpClosable: true,
+  },
+
+  tags: ["dev", "!autodocs", "!test"],
```

Comment:
That's all that I have found about hiding/showing stories in respective modes. There are 3 tags, but enabling `dev` mode still shows the ErrorForm story in [PR's SB link](https://kiwicom-orbit-sarka-storybook-migration-b2.surge.sh/). In case I disabled the dev mode, the story is not visible when running localhost:6007 either. 🙈